tests/test_api.py

```diff
@@ -75,7 +75,7 @@ def test_get_page(self) -> None:
     def test_get_attachment(self) -> None:
         with ConfluenceAPI() as api:
             data = api.get_attachment_by_name(
-                TEST_PAGE_ID, "figure_interoperability.png"
+                TEST_PAGE_ID, "figure/interoperability.png"
```

Comment:
As far as I recall, this was fixed on the branch `master`, and names are now sanitized to replace characters problematic for Confluence with `_`.